### PR TITLE
CD-359 Accessibility fixes

### PR DESCRIPTION
**Pull request by left23**
## Types of changes
:heavy_check_mark: Bug fix (non-breaking change which fixes an issue)

## Description
Fixes issues discovered running Accessibility tests via the Check1st service and augmented with Firefox Accessibility tool and Chrome Lighthouse testing.

## Motivation and Context
There are multiple issues included in this PR since it's typically a small fix per issue. From Epic CD-359

## Expected behavior
We should see better results, higher score, fewer errors when running accessibility tests

## Actual behavior
See the CD JIRA issues for screenshots or descriptions of current accessibility issues
  
## Impact
Depending on if/how the sub theme is overriding some element, some extra work may be needed to implement these fixes. Updating the base theme with these fixes included should not negatively impact anything.

## PR Checklist

- [x] I have included a CHANGELOG entry with the PR.
- [x] I have run local tests and the tests pass.
- [x] I have linted my code locally.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have made changes to the sub theme to reflect those in the base theme

### CODE REVIEW

**Comment by left23** [comment]
@rupl This is ready for review at your leisure. I'm happy to elaborate on or discuss any of it.

File: templates/navigation/menu--main.html.twig [hunk 1]
```diff
@@ -57,9 +57,10 @@
           Progressive enhancement: make sure there is always a menu entry.
           If the menu item has children and javascript is enabled then this will
           be replaced with a button to show the child menu.
+          If the menu item has children, an id attribute and value is added.
         #}
         {% if item.url %}
-        <a href="{{ item.url }}" id="{{ id }}"><span>{{ title }}</span></a>
+        <a href="{{ item.url }}"{% if item.below %} id="{{ id }}"{% endif %}><span>{{ title }}</span></a>
```
[the remark this answers is not in the document]
Comment (orakili) [reply templates/navigation/menu--main.html.twig:63]: @left23 Good catch, thanks. Totally my mistake.

I've added a fix for the ID hierarchy here: https://github.com/UN-OCHA/common_design/pull/318 (it's against this accessibility branch). 
 
I tested it on GHO-2022 (note: needed to update its subtheme `menu--main.html.twig` template override).

[the remark this answers is not in the document]
Comment (left23) [reply templates/navigation/menu--main.html.twig]: Thanks, I merged your PR  @orakili

**Comment by rupl** [comment]
For the record, I consider using conventional commits to be equivalent to a CHANGELOG entry. Checked the box for ya ;)

File: common_design_subtheme/templates/cd/cd-footer/cd-social-menu.html.twig [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
   {% block social_media %}
 
-  <a href="###" class="cd-footer-social__link">
+  <a href="https://www.facebook.com/UNOCHA" class="cd-footer-social__link">
```
Comment (rupl) [inline common_design_subtheme/templates/cd/cd-footer/cd-social-menu.html.twig:5]: Nice. I forget these pretty much every time. Great to have sensible defaults.

**Review by rupl — APPROVED** [review]
A lot of the changes are recognizable so I feel like running this tool on CD every once in a while is indeed a nice way to "nip it in the bud" before a small problem propagates across many theme implementations.

**Pull request merged**